frontend-opal/__tests__/components/Header/Header.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Header from '../../../components/Header'
+
+import renderer from 'react-test-renderer';
+
+describe('<Header />', () => {
+    it('renders correctly', () => {
```

Comment:
@Yuan-EPAM does this pass the linter? indentation is 2 spaces in the rest of the filest